### DISCUSSION TITLE
Batch processing and the order in which it faceswaps

### DISCUSSION THREAD

**Discussion by lebakerino** [glucauze/sd-webui-faceswaplab#121] [Q&A]
Hi, first off, great tool, a lot more options than roop and despite it sometimes missing faces unlike roop there is one issue I have with this.

When I say have a batch count of more than 1 image, sometimes I will cancel it and start again if I want to change something, so I may have 10 plus images processing. With Roop the way it processed was image by image, so if I cancelled at image 6, I'd  have 5 processed images, my issue with faceswaplab is it seems to be doing all the processing after all the images are processed, why? Is this an over sight? I've looked for an option regarding this and can't seem to find anything.

If it is simple to implement can there be an option that works like roop and it applies the faceswap process individually to each image when using batch count to generate more than 1 image.

Thanks.

**Comment by LucianoDaluz** [comment]
To stop havin it missing faces turn the NSFW filter off by slide it all the way to 1
